src/inc/FastConv.h

```diff
@@ -50,6 +52,37 @@ class CFastConv
     Error_t flushBuffer(float* pfOutputBuffer);
 
 private:
+    bool m_bIsInitialized = false;
+    int m_iBlockLength;
+    int m_iBlockNum;
+    int m_iCirIdx;
+    int m_iLengthOfIr;
+    int m_iReset;
+    float** m_pfIRMatrix;
+    float* m_pfInputBlock;
+    float* m_pfOutputBlock;
+    float* m_pfBlockConvOuput;
+    float* m_pfConvOuput;
+    CFft::complex_t *m_pfFreqInput;
+    CFft::complex_t *m_pfFreqIr;
+    CFft::complex_t *m_pfFreqConv;
+    float *m_pfRealInput;
+    float *m_pfImagInput;
+    float *m_pfRealIr;
+    float *m_pfImagIr;
+    float *m_pfRealConv;
+    float *m_pfImagConv;
+
+    float *m_pfTimeConv;
+    float *m_pfTimeInput;
+    float *m_pfTimeIr;
```

Comment:
Many of the member variables here are unique to either time domain or fft convolution. Consider using two different class for these two different modes.